src/Talis/EchoClient/Base.php

```diff
@@ -71,7 +71,7 @@ protected function getCacheClient()
      */
     protected function getStringSizeInBytes($input)
     {
-        return strlen(utf8_decode($input));
+        return strlen($input);
```

Comment:
This is used to limit the size of an event batch. However, it currently allows more than it should.

As per [documentation](https://www.php.net/manual/en/function.strlen.php):

> **strlen()** returns the number of bytes rather than the number of characters in a string.

Example where previous method would be very wrong:

```console
$ php -r 'var_dump(utf8_decode("🤨"));'
string(1) "?" // can't represent in ISO-8859-1 so swap with "?"

$ php -r 'var_dump(strlen(utf8_decode("🤨")));'
int(1) // 1 character ❌

$ php -r 'var_dump(strlen("🤨"));' 
int(4) // 4 bytes ✅
```

If we want the old behaviour (limit based on number of characters), but with correct implementation, we should then use `mb_strlen` function instead.